src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator.java

```diff
@@ -119,25 +122,34 @@ public String getApiUri() {
 
     @Override public void visitSources(SCMSourceObserver observer) throws IOException, InterruptedException {
         TaskListener listener = observer.getListener();
-        if (repoOwner.isEmpty()) {
-            listener.getLogger().format("Must specify user or organization%n");
-            return;
-        }
```

Comment:
:ant: makes no sense to move this *after* a network check when we can guarantee it will fail